BareBones Logistics Page

TyGriffin115 opened the pull request:
There is one bug where I could not figure out how to move the copyright logo to the bottom of the screen. Once that is fixed this should be done. Also I started working on this before the text editors were switched so I am not sure if that will cause a conflict.

TyGriffin115 committed:
BareBones Logistics Page

There is one bug where I could not figure out how to move the copyright logo to the bottom of the screen. Once that is fixed this should be done.

TyGriffin115 committed:
Merge branch 'master' into 76-Logistics-component

TyGriffin115 commented:
I merged the master branch into this branch and that fixed the problem with the text editors that I was having

Enoch-Hsiao reviewed (changes requested):
Instead of "ch", I would use 100% width for the textboxes. Some CSS tweaking is needed.

![Logistics](https://user-images.githubusercontent.com/51762139/95404679-aa23bf80-08e3-11eb-88eb-1dfa3bfb93ff.gif)

TyGriffin115 committed:
Removed copyright symbol

Enoch-Hsiao commented:
Also if you can remove unused imports, that would be greatly appreciated

TyGriffin115 committed:
FIxed bug for buttons

I changed the min-height to 600 instead of 500 which fixed the problem of buttons overlapping text. I also removed ch and used 100% for width instead.

TyGriffin115 committed:
Removed Unused Imports

Enoch-Hsiao committed:
css changes

Enoch-Hsiao committed:
Merge branch 'master' into 76-Logistics-component
